test/libyul/yulOptimizerTests/fullSuite/stack_compressor_msize.yul

```diff
@@ -1,15 +1,20 @@
 {
 	let a
 
+	function f() -> out
+	{
+		out := extcodesize(address())
+	}
+
 	function foo_singlereturn_0() -> out
 	{
-		mstore(lt(or(gt(1,or(or(gt(or(or(or(1,gt(or(gt(or(or(keccak256(pc(),or(gt(not(pc()),1),1)),1),not(1)),pc()),1),pc())),lt(or(1,sub(pc(),1)),1)),pc()),1),1),gt(not(pc()),1))),1),1),1)
-		sstore(not(pc()),1)
+		mstore(lt(or(gt(1,or(or(gt(or(or(or(1,gt(or(gt(or(or(keccak256(f(),or(gt(not(f()),1),1)),1),not(1)),f()),1),f())),lt(or(1,sub(f(),1)),1)),f()),1),1),gt(not(f()),1))),1),1),1)
+		sstore(not(f()),1)
 	}
 
 	function foo_singlereturn_1(in_1, in_2) -> out
 	{
-		extcodecopy(1,msize(),1,1)
```

Comment:
Please keep the msize, it switches the optimizer into a different mode (see the name of the test).